napari/utils/io.py

```diff
@@ -104,15 +104,24 @@ def imsave_tiff(filename, data):
         # 'compression' kwarg since 2021.6.6; we depend on more recent versions
         # now. See:
         # https://forum.image.sc/t/problem-saving-generated-labels-in-cellpose-napari/54892/8
+        # Deprecation warning for tuple instead of 'compressionargs' in 2022.7.28 version
         try:
-            tifffile.imwrite(filename, data, compression=('zlib', 1))
+            if tifffile.__version__ >= '2022.7.28':
+                tifffile.imwrite(filename, data, compressionargs={'zlib': 1})
+            else:
+                tifffile.imwrite(filename, data, compression=('zlib', 1))
```

Comment:
```suggestion
            from importlib.metadata import version
            from packaging.version import parse as parse_version 
            
            if parse(version("tifffile")) >= parse('2022.7.28'):
                tifffile.imwrite(filename, data, compressionargs={'zlib': 1})
            else:
                tifffile.imwrite(filename, data, compression=('zlib', 1))
```

With moving import to top level. 

String comparison is not a safe way to check, for example the `2022.10.10` is older than `2022.7.28` in string comparison.

Reply:
Thanks for the review and suggestion. Good point on the comparison.